agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -344,3 +353,100 @@ func TestTwoTasksSharedLocalVolume(t *testing.T) {
 	rExitCode := rTask.Containers[0].ExitCode
 	assert.NotEqual(t, 42, rExitCode, fmt.Sprintf("Expected exit code of 42; got %d", rExitCode))
 }
+
+func TestGMSAFile(t *testing.T) {
+	RequireDockerAPIVersion(t, ">=1.30")
+
+	agentOptions := AgentOptions{
+		ExtraEnvironment: map[string]string{
+			"ZZZ_SKIP_DOMAIN_JOIN_CHECK_NOT_SUPPORTED_IN_PRODUCTION": "true",
+		},
+	}
+	agent := RunAgent(t, &agentOptions)
+	defer agent.Cleanup()
+
+	agent.RequireVersion(">=1.26.0")
```

Comment:
It is supposed to be agent.RequireVersion(">=1.33.0") for now, and once we release agent, it needs to be updated to 1.34.0.

Reply:
Done